modelbaker/db_factory/pg_command_config_manager.py

```diff
@@ -79,18 +79,24 @@ def get_uri(self, su: bool = False, qgis: bool = False) -> str:
             uri += ["dbname='{}'".format(self.configuration.database)]
 
         # only provide authcfg to the uri when it's needed for QGIS specific things
-        if (
-            qgis
-            and self.configuration.dbauthid
-            and (
-                not service_config
-                or not (
-                    service_config.get("user", None)
-                    and service_config.get("password", None)
-                )
+        if self.configuration.dbauthid and (
+            not service_config
+            or not (
+                service_config.get("user", None)
+                and service_config.get("password", None)
             )
         ):
-            uri += ["authcfg={}".format(self.configuration.dbauthid)]
+            if qgis:
+                # only provide authcfg to the uri when it's needed for QGIS specific things
+                uri += ["authcfg={}".format(self.configuration.dbauthid)]
+            else:
```

Comment:
Not sure, but is authcfg possible without qgis?

Reply:
I guess so. Maybe the confusion arises because of the flag name (`qgis`)?
Not sure when `qgis=True` is used. In your opinion, what name could be more appropriate?

Reply:
ah now I get it. 

no you can leave it "qgis"